kiali/kiali-ui
# Add AuthorizationPolicy form

This PRs provides a new form to create AuthorizationPolicy objects.

Note:
- IstioConfig new form only support the creation of the object, updates are done via yaml editor.
- AuthorizationPolicy is a complex object with multiple edges and corner cases, hard to focus just on some, so, the form supports all cases (in other objects as Gateways/Sidecars forms only support a subset of most direct scenarios).
- User requires to catch up with the terminology used by AuthorizationPolicy entity.

How to test:
- Best strategy would be to follow the Istio tasks tutorial https://istio.io/docs/tasks/security/authorization/ and check that all scenarios can be done via the form, checking that resulting yaml is valid and well formed.

Fixes https://github.com/kiali/kiali/issues/2041

It requires backend PR https://github.com/kiali/kiali/pull/2680
